consumer/src/test/kotlin/integrations/kafka/KafkaEventSinkPattern.kt

```diff
@@ -0,0 +1,90 @@
+package integrations.kafka
+
+import kotlinx.coroutines.runBlocking
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.ByteArraySerializer
+import org.hamcrest.Matchers
+import org.junit.Test
+import org.neo4j.function.ThrowingSupplier
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.test.assertion.Assert
+import streams.serialization.JSONUtils
+import java.util.*
+import java.util.concurrent.TimeUnit
+import kotlin.test.assertEquals
+
+class KafkaEventSinkPattern : KafkaEventSinkBase() {
```

Comment:
This is a test as well right? `IT` suffix would make it clearer I think

Reply:
As well as above